esque/cli/commands.py

```diff
@@ -273,7 +273,8 @@ def edit_topic(state: State, topic_name: str):
 @click.option("--offset-by-delta", help="Shift offset by specified value", type=click.INT, required=False)
 @click.option(
     "--offset-to-timestamp",
-    help="Set offset to the value closest to the specified message timestamp in the format YYYY-MM-DDTHH:mm:ss (NOTE: this can be a very expensive operation).",
+    help="Set offset to the value closest to the specified message timestamp in the format YYYY-MM-DDTHH:mm:ss "
```

Comment:
This will not actually put you to the closest value but to the first one at or after that timestamp.
If it were the closest one we'd have to check for the absolute difference in time between two messages surrounding that timestamp.

Reply:
Maybe say something like "Skip over all messages before that timestamp"...